src/lib/message.ts

```diff
@@ -30,7 +32,8 @@ class GleeMessage extends EventEmitter {
   private _outbound: boolean
   private _cluster: boolean
   private _params: { [key: string]: string }
-  
+  private _query: any
```

Comment:
Any? I think it's pretty safe to assume that the type would be something like `{ [key: string]: string } | { [key: string]: string[] }`. All the query parameters are going to be received as strings independently of their initial type.

Let's try to make a bigger effort here instead of just throwing an `any` type, please.